res/sections/25-ChallengedNets.tex

```diff
@@ -16,12 +15,11 @@ \section{Challenged Networks}
 
 Due to packet losses, dupAcks, timeouts etc \dots the link is never used
 to his full potential.
-Disruptive/intermitted links (con interferenze) : with this kind of links
```

Comment:
@Polpetta please do not erase my italian hints 🙏 . In some cases they are useful to me! 🙏  🙏 .

Reply:
Okok, is a problem to add the hints as a footnote? :blush: